cmd/registry.go

```diff
@@ -94,16 +122,32 @@ func updateCachedRegistries(regList []Registry) error {
 	return nil
 }
 
-func addRegistry() {
+func addRegistry(regType string) {
 	var regList []Registry
 	err := viper.UnmarshalKey("Registries", &regList)
 	if err != nil {
 		fmt.Println("Error unmarshalling config: ", err)
 		return
 	}
+
+	// TODO: Add all cases here. For simplicity I'm keeping it like this until the v2 work merges
+	switch regType {
+	case "dockerhub":
+		registryConfig.Config = defaultDockerHubConfig
+	case "local_openshift":
+		registryConfig.Config = defaultLocalOpenShiftConfig
+	case "helm":
+		registryConfig.Config = defaultHelmConfig
+	default:
+		fmt.Printf("Unrecognized type of registry [%v]\n", regType)
```

Comment:
For users who haven't taken a look at source code or docs, it's unclear what my options are for types of registries. We should include some examples of acceptable types on this failure condition.